### PR TITLE
Tab fix

### PR DESCRIPTION
**Pull request by JuliaWinn**
Fixed bug listed here: https://github.com/18F/analytics.usa.gov/pull/211

Moved Google Analytics events related to the tab navigation into the aria code that was previously handling the events.

### CODE REVIEW

File: _development.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ url: http://localhost:4000
 #   analytics --output=[dir]
 # and running from [dir]:
 #   serve
-data_url: http://localhost:3000
+data_url: http://localhost:3000/
```
Comment (konklone) [inline _development.yml]: Was this change necessary for you in development? The `data_url` fields we use don't expect it to end with a trailing slash (you can see some examples of this below in the rest of the PR diff).

**Comment by konklone** [comment]
It looks like this PR accidentally included https://github.com/18F/analytics.usa.gov/commit/d3bf0ec3e200f7e041514e31d8f75ba8c976a877 as well, and it looks like https://github.com/18F/analytics.usa.gov/commit/d2d9a9072d0bc4c2b9203a6d430b400d09f10144 has a number of unrelated things going on. Since the PR also doesn't perfectly solve #224 (since now the URL doesn't change), I think I'd like to close this in favor of a new one following @shawnbot's suggestions in https://github.com/18F/analytics.usa.gov/issues/224#issuecomment-149641713 (a scoped event listener).

**Comment by JuliaWinn** [comment]
Shawn said the URL shouldn't need to change and the fact that it was
earlier was a sign that the listener wasn't firing.

**Comment by konklone** [comment]
Fixed in #226.
